submissions/urls.py

```diff
@@ -1,11 +1,12 @@
+""" Submissions api urls. """
 from __future__ import absolute_import
 
-from django.conf.urls import patterns, url
```

Comment:
Given that this function was removed in Django 1.10, I don't think this file is even in use (here or in edx-platform).  The code below is still slightly wrong, but it should be safe to just delete the whole file.